homeassistant/components/fjaraskupan/strings.json

```diff
@@ -0,0 +1,20 @@
+{
+  "title": "Fj\u00e4r\u00e5skupan",
```

Comment:
No need to specify since it's same as manifest. It will be automatically set.

Reply:
I think this was added by the scaffold.

Reply:
https://github.com/home-assistant/core/pull/53350